include/stdexec/execution.hpp

```diff
@@ -5848,10 +6139,133 @@ namespace stdexec {
+            return get_env(__self.__receiver_);
+          }
+
+          friend void tag_invoke(start_t, __operation& __self) noexcept {
+            (*__get_vtable(__self.__storage_)->__start_)(__get_object_pointer(__self.__storage_));
+          }
+        };
+
+      template <class _Sigs, class _ReceiverQueries>
+        struct __sender_vtable {
+          using __receiver_ref_t = __receiver_ref<_Sigs, _ReceiverQueries>;
+          __unique_operation_storage (*__connect_)(void*, __receiver_ref_t);
+          __unique_operation_storage operator()(connect_t, void* __sender, __receiver_ref_t __receiver) const {
+            return __connect_(__sender, __receiver);
+          }
+        };
+
+      template <class _Sender, class _ReceiverQueries>
+      struct __sender_vtable_fn {
+        using _Sigs = completion_signatures_of_t<_Sender>;
+        using __receiver_ref_t = __receiver_ref<_Sigs, _ReceiverQueries>;
+        constexpr __unique_operation_storage
+        (*operator()() const noexcept)(void*, __receiver_ref_t) {
+          return +[](void* __object_pointer, __receiver_ref_t __receiver) -> __unique_operation_storage {
+            _Sender& __sender = *static_cast<_Sender*>(__object_pointer);
+            static_assert(receiver_of<__receiver_ref_t, _Sigs>);
+            using __op_state_t = connect_result_t<_Sender, __receiver_ref_t>;
+            return __unique_operation_storage{std::in_place_type<__op_state_t>, 
+                __conv{[&] { return connect((_Sender&&)__sender, (__receiver_ref_t&&)__receiver); }}};
+          };
+        }
+      };
+
+      template <class _Sender, class _Queries>
+        inline constexpr __sender_vtable<completion_signatures_of_t<_Sender>, _Queries>
+        __sender_vtbl{__sender_vtable_fn<_Sender, _Queries>{}()};
+
+      template <class _Sigs, class _Queries, class _Sender>
+        constexpr const __sender_vtable<completion_signatures_of_t<_Sender>, _Queries>*
+        tag_invoke(__create_vtable_t<__sender_vtable<_Sigs, _Queries>>, __mtype<_Sender>) noexcept {
+          return &__sender_vtbl<_Sender, _Queries>;
+        }
+
+      template <class _Sigs, class _ReceiverQueries, class _SenderQueries>
+        class __sender {
+         public:
+          using completion_signatures = _Sigs;
+
+          template <sender _Sender>
+            __sender(_Sender&& __sender)
+              : __storage_{(_Sender&&) __sender} {}
+
+         private:
+          using __vtable_t = __sender_vtable<_Sigs, _ReceiverQueries>;
+
+          __storage_t<__unique_storage<>, __vtable_t> __storage_;
+
+          template <receiver_of<_Sigs> _Rcvr>
+              requires sender_to<__sender, _Rcvr>
+            friend __operation<__sender, std::decay_t<_Rcvr>, _ReceiverQueries> 
+            tag_invoke(connect_t, __sender&& __self, _Rcvr&& __rcvr) {
+              return {(_Rcvr&&) __rcvr, (__sender&&) __self};
+            }
+        };
+    } // namespace __sender
+  } // namepsace __any
```

Comment:
I just realized I started this work in the wrong place. It can't be in `stdexec::`. Could you move it to `exec/any_sender_of.hpp`, or something?

Reply:
Yes I will do so